manifests/params.pp

```diff
@@ -35,7 +35,7 @@
   $user = 'qpidd'
   $group = 'qpidd'
 
-  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python2-qpid']
+  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client']
```

Comment:
You'd almost thing we just need qpid-cpp-server, but in this module we probably need the client package for queue actions.